tests/data_context/fixtures/contexts/incomplete_uncommitted/great_expectations/great_expectations.yml

```diff
@@ -6,7 +6,7 @@
 #   - Read our docs: https://docs.greatexpectations.io/en/latest/reference/data_context_reference.html#configuration
 #   - Join our slack channel: http://greatexpectations.io/slack
 
-config_version: 2
```

Comment:
I already updated these files quite a bit with other changes so in actuality, they already conform to `config_version: 4` without additional changes